test/Mono.Linker.Tests.Cases/Substitutions/FeatureSubstitutionsInvalid.xml

```diff
@@ -6,12 +6,8 @@
         <method signature="System.Boolean BooleanFeatureMethod()" body="stub" value="false" feature="BooleanFeature" featurevalue="true" />
       </type>
       <type fullname="Mono.Linker.Tests.Cases.Substitutions.FeatureSubstitutionsInvalid/Foo">
-        <field signature="NonExistentField" />
+        <field name="NonExistentField" />
         <method signature="NonExistentMethod" />
-        <event signature="NonExistentEvent" />
```

Comment:
Why did you remove these? They seem like good tests to have.

Reply:
BodySubstituterStep does not have any code to handle events, properties nor field signatures. All of these were just warning because of the ResolveXmlStep

Reply:
I see - makes sense.